src/library/scala/collection/immutable/NumericRange.scala

```diff
@@ -189,7 +189,7 @@ extends AbstractSeq[T] with IndexedSeq[T] with Serializable {
       import numAsIntegral._
       if (isEmpty) num fromInt 0
       else if (numRangeElements == 1) head
-      else ((num fromInt numRangeElements) * (head + last) / (num fromInt 2))
+      else (num fromInt numRangeElements) * ((head + last) / (num fromInt 2))
```

Comment:
Yes, I noticed that afterwards, but that formula isn't quite right either since (head + last) might overflow.  So you have to do the division separately, and then add, and then add one if both are odd, and all that is not a good idea for types that don't overflow like BigInt so I handle those separately.  (Once I get the new PR in.)